config.py

```diff
@@ -138,4 +138,9 @@ class Config(object):
             ReturnCodes.TEMPORARY,
             (False, False)
         ),
+        'postcrimes.org': (

```

Comment:
please also add in www.postcrimes.org

if you're up for a small refactor and have an idea for a solution, we could try and make it less verbose to specify the same redirect for multiple domains.

Reply:
(maybe nust file that as a bug, and we'll go from there)